### ISSUE TITLE
Does not save on nextcloud

### ISSUE THREAD

**Issue by Noxmiles** [derbenx/nextcloud-familytree#2]
When I open the app it's always empty. I only can import the exported json file. It doesn't save within the app.

Also, clicking on the json file within files does not give the possibility to open it in/with this app.

**Comment by derbenx** [comment]
Yup, when you open apps they typically are empty until you load the data. ;) Especially confidential files containing personal info. The app has a save and a load option, like all other apps that deal with data. While the app works on mobile, it's more for use with a PC. The nextcloud file sync program gives you a folder on your PC to sync with nextcloud, saving and loading to/from there will work nicely.

At this moment, I do not know how to save directly to nextcloud from the app. This wasn't a feature I was considering yet. Properly sorting GED files is where I am researching now.

Lastly, you don't want json files associated with a tree reader. Json files are used for storing all kinds of data. What you want to do requires an unique save extension name.

**Comment by derbenx** [comment]
@eltos
It's open source, feel free to fix whatever you don't like... ;)
